setup.py

```diff
@@ -1,15 +1,22 @@
+from __future__ import absolute_import
 from setuptools import setup, find_packages
 from oauth_provider import __version__ as version
 import os
 
+
 def strip_comments(l):
     return l.split('#', 1)[0].strip()
 
+
 def reqs(*f):
-    return list(filter(None, [strip_comments(l) for l in open(
-        os.path.join(os.getcwd(), *f)).readlines()]))
+    return list([_f for _f in [strip_comments(l) for l in open(
+        os.path.join(os.getcwd(), *f)).readlines()] if _f])
+
 
-install_requires = reqs('requirements.txt')
+install_requires = [
+    'Django>=1.3,<=1.9',
+    'python-oauth2'
```

Comment:
The name of the upstream package is 'oauth2'.  That's what you'll want here.

Reply:
@feanil did so many experiments. it was the egg name in previous attempt.